### ISSUE TITLE
*/int syntax doesn't appear to work

### ISSUE THREAD

**Issue by pansapiens** [ahawker/crython#29]
Test script:

```
#!/usr/bin/env python
import crython
import time

@crython.job(second='*/3')
def foo():
    print("I'm using */3 syntax.")

@crython.job(second=range(0, 60, 2))
def bla():
    print("I'm using range(0, 60, 2).")

if __name__ == '__main__':
    crython.start()
    try:
        while True:
            time.sleep(0.2)
    except KeyboardInterrupt:
        pass

    print("Done")
```

Prints:
```
I'm using range(0, 60, 2).
I'm using range(0, 60, 2).
I'm using range(0, 60, 2).
... etc ...
```

I would expect "`I'm using */3 syntax.`" to be printed every three seconds also, but it's not.

Is this a bug or am I doing something wrong ?

Latest master and pypi 0.0.4 version.

**Comment by ahawker** [comment]
@pansapiens Thanks for reporting this and I apologize for the issue!

I tracked this down to a bug with some of the compatibility code I added to support py3+.

This should be in #30 .

**Comment by ahawker** [comment]
Merged and deployed. Fix should be available in version `0.0.5` out in `pypi`.
